main.go

```diff
@@ -168,11 +168,15 @@ func Copy(client *ssh.Client) {
 }
 
 // ConfigureAuthentication configures the authentication method.
-func ConfigureAuthentication(key string, password string) []ssh.AuthMethod {
+func ConfigureAuthentication(key string, passphrase string, password string) []ssh.AuthMethod {
 	// Create signer for public key authentication method.
 	auth := make([]ssh.AuthMethod, 1)
 	if key != "" {
-		targetSigner, err := ssh.ParsePrivateKey([]byte(key))
+		if passphrase != "" {
+			targetSigner, err := ParsePrivateKeyWithPassphrase([]byte(key), []byte(passphrase))
```

Comment:
@Briaker Is there an `ssh.` missing here?

Reply:
Fixed it

Reply:
Thank you! I am just fixing the CI pipeline so we can verify that this compiles and build the container.